index.html

```diff
@@ -199,5 +191,40 @@ <h1 class="about-title">About me</h1>
                 </div>
         </div>
     </section>
+    <section class="contact">
+        <div class="cont">
+            <h3 class="contact-text">I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h2>
+            <form action="https://formspree.io/f/xvoyeaqj" method="post">
+                <ul class="forms">
+                    <li>
+                        <input type="text" name="name" class="input" id="name" placeholder="Full name" required>
+                        <label for="name" class="label">Full name</label>
+                    </li>
+                    <li>
+                        <input type="email" name="email" class="input" id="email" placeholder="Email" required>
+                        <label for="email" class="label">Email</label>
+                    </li>
+                    <li> 
+                        <textarea class="area" rows="5" name="message">Enter text here</textarea>
+                    </li>
```

Comment:
- [ ] Compared to the image the previous reviewer gave, there is not any difference with the actual design, you are recommended to implement all the required changes before you resubmit the work, well I will try to highlight each part as I did above, on this image below, take a look at the text-area, the is a background, yes on your too, but the color does not match the one on the original design, what do you think if you change that ab it?
- [ ] Your placeholder is too black more than the one in the original design, it is looking like the actual input data you see, please take a look closer and modify it.

- [ ] Your `get in touch` button has round corners,... hum.. if I believe my eyes, this is not how it should look compared to the original design, isn't it?.... So please consider making a few touches on that too.

<img width="774" alt="p2" src="https://user-images.githubusercontent.com/77843050/205963523-e2b15f6d-ee70-4eae-bd01-b5de045f54e6.png">